### ISSUE TITLE
关于dropout

### ISSUE THREAD

**Issue by ucasiggcas** [ZiyaoGeng/Recommender-System-with-TF2.0#62]
hi，大佬，[这行代码](https://github.com/ZiyaoGeng/Recommender-System-with-TF2.0/blob/master/Caser/model.py#L88)如果没有设置training=True，dropout发挥不了作用，测试时去掉。
详见[我的博文](https://iggcas.blog.csdn.net/article/details/121696915)。

**Comment by ZiyaoGeng** [comment]
你好，关于这个问题你可以看一下[官方API](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Dropout)，在fit的时候training会自动判断
`When using model.fit, training will be appropriately set to True automatically`
我在[Issue#4](https://github.com/ZiyaoGeng/Recommender-System-with-TF2.0/issues/4)也提过。
